app/models/mixins/provider_proxy_mixin.rb

```diff
@@ -0,0 +1,13 @@
+module ProviderProxyMixin
+  extend ActiveSupport::Concern
+
+  def http_proxy_uri
+    proxy = VMDB::Util.http_proxy_uri("#{emstype}_http_proxy".to_sym)
```

Comment:
How about following the suffix scheme like refresh and others
https://github.com/ManageIQ/manageiq/blob/faec76397e8e77d65e98432d8d46e38357632832/config/settings.yml#L1100

also I would go with the `defaults` like [here](https://github.com/ManageIQ/manageiq/blob/faec76397e8e77d65e98432d8d46e38357632832/config/settings.yml#L1101)

but I'm not sure about the implications of changing the structure of settings? @blomquisg ?